test/test_commands.py

```diff
@@ -103,6 +103,12 @@ def test_log(self):
         expected = get_expected_output('log_limit')
         self.assertEqual(output, expected)
 
+    def test_log_mergeonly(self):
```

Comment:
Nitpick: please add a dash between `merge` and `only` (as in the option).

Reply:
I assume you meant underscore. I agree and updated.